web/v2/backend/src/main/java/com/tourneygen/web/Models/DTOs/TeamDTO.java

```diff
@@ -0,0 +1,121 @@
+package com.tourneygen.web.Models.DTOs;
+
+import com.google.gson.Gson;
+import com.tourneygen.web.Models.Team;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+public class TeamDTO {
+
+  private Long id;
+
+  @NotNull private Set<String> roster;
+
+  private int wins = 0;
+
+  private int losses = 0;
+
+  @NotNull private long owner;
+
+  @NotBlank private String name;
+
+  @NotBlank private String description;
+
+  @NotNull private long league;
+
+  public TeamDTO() {}
+
+  public TeamDTO(Team team) {
+    this.id = team.getId();
+    this.roster = team.getRoster();
+    this.wins = team.getWins();
+    this.losses = team.getLosses();
+    this.owner = team.getOwner().getId();
+    this.name = team.getName();
+    this.description = team.getDescription();
+    this.league = team.getLeague().getId();
+  }
+
+  public static List<TeamDTO> findAll(List<Team> teamList) {
+    ArrayList<TeamDTO> teamDTOS = new ArrayList<>();
+    for (Team team : teamList) {
```

Comment:
`teamList.stream().forEach(team -> teamDTOS.add(new TeamDTO(team))` ? 

I'm not 100% yet on this whole streaming thing, but I think loops with one call in them are a good candidate for using the steam().forEach

maybe?

Reply:
il try it

Reply:
Merci

Reply:
fancy coding implemented